README.md

```diff
@@ -1,46 +1,54 @@
 
-## Files in this repository
+## B.C. Design Tokens
 
-```
-docs/           - Project Documentation
-└── images        
-└── icons         
+The [B.C. Design Tokens package](https://github.com/bcgov/design-system/tree/docs/packages/design-tokens) is the first release as part of the new design system.
 
-openshift/      - OpenShift-specific files
-├── scripts     - helper scripts
-└── templates   - application templates
-```
+Design tokens provide a method to quickly and consistently implement the province's look and feel for digital services. They offer a set of standardized options to help developers style user interfaces in a consistent and future-proof way.
 
-## Deployment (Local Development)
+* [Installable package on npm](https://www.npmjs.com/package/@bcgov/design-tokens)
+* [Source code](https://github.com/bcgov/design-system/tree/master/packages/design-tokens)
+* [Figma library](https://www.figma.com/community/file/1326994583954765832)
 
-* Developer Workstation Requirements/Setup
-* Application Specific Setup
+## Component library
 
-## Deployment (OpenShift)
+The component library will provide an inventory of reusable UI components, including:
 
-See (openshift/Readme.md)
+* Components in Figma
+* Reference implementations in React
+* Support for other languages and frameworks is currently out-of-scope.
 
-## Getting Help or Reporting an Issue
+The component library is currently in active development.
 
-To report bugs/issues/feature requests, please file an [issue](https://github.com/BCDevOps/opendev-template/issues/).
+A [proof-of-concept/preview](https://www.npmjs.com/package/@bcgov/design-system-react-components) of the React library is published on npm.
 
-## How to Propose a Component
+## Contact
 
-If you would like to propose a component to the design system, please see our [Propose a Component](.github/ISSUE_TEMPLATE/propose-a-new-component.md) guideline.
+The B.C. Design System is maintained by Government Digital Experience (GDX), a division of the Ministry of Citizens' Services. You can contact the design system team by email at [designsystem@gov.bc.ca](mailto:designsystem@gov.bc.ca), or [open an issue](https://github.com/bcgov/design-system/issues/new/choose).
 
-Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). 
-By participating in this project you agree to abide by its terms.
+## Legacy documentation
+> [!WARNING]
+> The previous B.C. government design system is not actively supported. These resources are provided for reference only. These resources will be officially deprecated and archived after the new BCDS component library is released.
+
+Documentation and components for the legacy design system are accessible in this repo and on DevHub:
+
+* [Developer documentation](https://developer.gov.bc.ca/Design-System/About-the-Design-System)
+* [Legacy components on DevHub](https://developer.gov.bc.ca/components)
```

Comment:
We should leave out this "Legacy components" link. It's pointing to a landing page for any document with the label "Components" in DevHub, which isn't limited to the Design System stuff. Example of how that label gets applied: https://github.com/bcgov/devhub-app-web/blob/master/web/topicRegistry/design-system.json#L54

Better to just link to that "Developer documentation" page above. When you include the `/Design-System` part of the path segment as you have done, you get the full left navigation with the list of Design System components there.

Reply:
Good catch — I've just removed that link.